### PR TITLE
INDY-156 POOL_UPGRADE txn unclear error message

### PR DESCRIPTION
**Pull request by hadleym**
Added logic to sovrin_node/server/config_req_handler.py
When a POOL_UPGRADE request that has already been accepted is submitted, there will be a UnauthorizedClientRequest thrown now.

### CODE REVIEW

File: sovrin_node/server/config_req_handler.py [hunk 1]
```diff
@@ -63,6 +63,12 @@ def validate(self, req: Request, config=None):
             # present
             status = self.upgrader.statusInLedger(req.operation.get(NAME),
                                                   req.operation.get(VERSION))
+            if status == START and action == START:
+                raise UnauthorizedClientRequest(
```
Comment (ashcherbakov) [inline sovrin_node/server/config_req_handler.py]: Why do we raise `UnauthorizedClientRequest` here? Should we use some other type of exception? `UnauthorizedClientRequest` is usually raised when a sender doesn't have enough permissions (for example Role) to perform a request.

Comment (hadleym) [inline sovrin_node/server/config_req_handler.py]: Sure, we can make it a more appropriate message.  The ticket suggested UnauthorizedClientRequest, I will change and re-submit this morning.

**Comment by hadleym** [comment]
changed to InvalidClientRequest

File: sovrin_node/test/upgrade/test_pool_upgrade_reject.py [hunk 2]
```diff
@@ -42,4 +42,10 @@ def testNonTrustyCannotCancelUpgrade(looper, validUpgradeSent,
     looper.run(eventually(checkRejects, stClient, req.reqId,
                           'cannot do'))
 
+def test_accept_then_reject_upgrade(looper, trustee, trusteeWallet, validUpgradeSent, validUpgrade):
+    validUpgrade2 = deepcopy(validUpgrade)
+    _, req = sendUpgrade(trustee, trusteeWallet, validUpgrade2)
+    timeout = plenumWaits.expectedReqNAckQuorumTime()
+    looper.run(eventually(checkRejects, trustee, req.reqId,
+                          'UnauthorizedClientRequest', retryWait=1, timeout=timeout))
```
Comment (ashcherbakov) [inline sovrin_node/test/upgrade/test_pool_upgrade_reject.py]: Should we change this to be `InvalidClientRequest`?

Comment (hadleym) [inline sovrin_node/test/upgrade/test_pool_upgrade_reject.py]: This was changed to InvalidClientRequest.

**Comment by hadleym** [comment]
Reposted PR in slack. Please let me know if this is not the correct work flow.

File: sovrin_node/test/upgrade/test_pool_upgrade_reject.py (hunk 2 above)
Comment (hadleym) [inline sovrin_node/test/upgrade/test_pool_upgrade_reject.py]: Ah, I was not correcting this properly.

**Pull request merged**